ledger/src/shred.rs

```diff
@@ -995,7 +995,7 @@ pub fn max_entries_per_n_shred(
     num_shreds: u64,
     shred_data_size: Option<usize>,
 ) -> u64 {
-    let data_buffer_size = ShredData::capacity(/*merkle_proof_size:*/ None).unwrap();
+    let data_buffer_size = ShredData::capacity(/*merkle_proof_size:*/ Some(6)).unwrap();
```

Comment:
I might be missing something, but does this need to change with this PR?

Reply:
```
broadcast_stage::standard_broadcast_run::test::test_slot_interrupt
```
fails without this change because `None` means legacy shreds but the test creates merkle shreds.

Reply:
Ahh I see, unit tests are run with `ClusterType::Development` so `should_use_merkle_variant(slot, cluster_type)` is changing from `false` to `true` in the context of unit tests.

And why specifically `Some(6)` ?

Reply:
The default 32:32 erasure batches will be 64 shreds, so the Merkle proof size would be `log2(64) == 6`:
https://github.com/solana-labs/solana/blob/6d30429dd/ledger/src/shred/merkle.rs#L1319

This is a test-only function so I didn't want to make it more complicated for now.

Reply:
Got it - I'm fine keeping the function simple and using hard-coded value.

That being said, would you mind throwing in a small comment above this line to explain why 6? Something as simple as your explanation above, ie:
```rust
// Default 32:32 erasure batches yields 64 shreds; log2(64) = 6
let merkle_proof_size = Some(6);
```

Reply:
done.